Bottom: f467de80c42c78d5e39a55942a1e66f0d1a7ffdc
Top:    0fc4c34d915b238af80ecef63afbe9153721a2cf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 01:15:18 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
